en/airframes/adding_a_new_frame.md

```diff
@@ -63,21 +67,34 @@ then
 	param set FW_RR_FF 0.6
 	param set FW_RR_P 0.04
 fi
+```
 
+Set frame type:
+```bash
 # Configure this as plane
 set MAV_TYPE 1
+```
+
+Set the mixer:
+```bash
 # Set mixer
 set MIXER wingwing
+```
+
+```bash
```

Comment:
@bkueng Is there anything we can say about this section below? (ie what is happening here)

Reply:
We can say that this configures the PWM outputs: which outputs we want to drive/activate, and the levels.